src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java

```diff
@@ -1122,4 +1123,61 @@ protected void initClassDefaults(final UIDefaults table) {
         };
         table.putDefaults(uiDefaults);
     }
+
+    /**
+     * Returns a new Focus Ring color by changing saturation
+     * and setting the brightness to 100% for incoming focusRingColor.
+     *
+     * If the incoming focusRingColor is equal to white/black/grayish,
+     * the returned focusRingColor is Light Gray. For all other colors,
+     * new focusRingColor (in the latter case), is obtained by adjusting
+     * the saturation levels and setting the brightness to 100% of the
+     * incoming focusRingColor.
+     *
+     * @param focusRingColor - the {@code Color} object
+     * @return the {@code Color} object corresponding to new HSB values
+     */
+    static Color deriveLighterFocusRing(Color focusRingColor) {
```

Comment:
I guess we are not generating a "lighter" focus ring now, just a prominent one, so I think this function needs renaming..

Reply:
Also, I see if we start off with "Graphite" accent color then the focus ring is "Graphite" but then even if we change to any other accent color, the focus ring remains "Graphite"

Reply:
@prsadhuk The following type properties where checked to for cellFocusRing color

- [alternateSelectedControlColor](https://developer.apple.com/documentation/appkit/nscolor/1533135-alternateselectedcontrolcolor) 
**Issue:** Doesn't follow accent color in macos 10.15 ad deprecated property.

-  [controlAccentColor](https://developer.apple.com/documentation/appkit/nscolor/3000782-controlaccentcolor) 
**Issue:** Causes build issues due to version compatibility, property supported on only macOS 10.14+

- [accentColor](https://developer.apple.com/documentation/swiftui/color/accentcolor) 
**Issue:** Property supported on only macOS 10.15+, compatibility and run time issues.

- [keyboardfocusindicatorcolor](https://developer.apple.com/documentation/appkit/nscolor/1532031-keyboardfocusindicatorcolor) 
**Issue:** Works well for most of the scenarios (including on-the-fly changes) and follows the accent color changes except when we start the application with accent color set to Graphite and then change accent color on-the-fly (as mentioned above). The cell focus ring remains gray. The issue seems to be with the system color returned by the type-property "**keyboardfocusindicatorcolor**" under this setting.

Reply:
@prrace @prsadhuk 
The issue with the system color returned by **keyboardFocusIndicatorColor** mentioned above, persists on macOS 12.3 as well.

Reply:
"accentColor" is swift property so not usable for jdk but you can use "controlAccentColor" which seems to work with 10.15 and 12.1 as per my limited testing.
You need to use @available check though..
Even though it may not work for version lesser than 10.14, I guess it's ok as those versions are more or less obsolete..
While you are it, please consider renaming this function as told.

Reply:
@prsadhuk Thank you for the suggestion. I will add in the required changes , test it and update the PR.

Reply:
@prsadhuk Update PR with the recommended changes. The "controlAccentColor" changes works as expected in Mac 12.3 as well. 
One thing that I wanted to clarify with the fix is - the focus ring color ("Focus.ring") is used at two other places AquaButtonUI and AquaFocus (for focused icon). With the current fix I'm setting **KEYBOARD_FOCUS_COLOR** as below - 

<img width="1092" alt="image" src="https://user-images.githubusercontent.com/95945681/162673487-26006ca1-9373-4fe7-8f0b-f99cd47b2eb7.png">

Before and After images of focused icon -

**Before**

<img width="185" alt="image" src="https://user-images.githubusercontent.com/95945681/162674337-ebbfea9f-e753-4b4c-a624-7cef5b8d8c1b.png">

**After (focus around icon slightly darker - focus ring color obtained from controlAccentColor property)**

<img width="198" alt="image" src="https://user-images.githubusercontent.com/95945681/162674444-b917de71-8e91-472e-a8f0-da48df0a43a5.png">

Please let me know in case I need to add a NEW color (as shown below) instead of reusing the KEYBOARD_FOCUS_COLOR if the color effects on focused icon/ button doesn't look correct.

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/95945681/162675955-efd9c77f-6d7b-41e7-803a-50f49154fba5.png">

Reply:
It seems KEYBOARD_FOCUS_COLOR via `focusRingColor` is also used for menuSelectedBackgroundColor which is used for 
```
CheckBoxMenuItem.selectionBackground
ComboBox.selectionBackground
Menu.selectionBackground
MenuBar.selectionBackground
MenuItem.selectionBackground
"PopupMenu.selectionBackground", menuSelectedBackgroundColor,
RadioButtonMenuItem.selectionBackground
```
which might also get affected if you use new property, so you need to test all those widgets if you intend to use reuse KEYBOARD_FOCUS_COLOR  so I guess safe bet is to use new color as you mentioned to limit the change to highlight color.

Reply:
@prsadhuk Following code changes were pushed and ready for re-review

- New color added to CSystemColor.m - CELL_HIGHLIGHT_COLOR, to limit the effects of highlight color changes
- method renamed from deriveLighterFocusRing() to deriveProminentFocusRing()
- Test case made headful to load system colors

Reply:
@prsadhuk With the new changes a public method is added to AquaImageFactory, does this require a CSR?

Reply:
If you are talking about getCellHighlightColorUIResource() then no. We don't have specs for this class.
I have added similar getSelectedControlColorUIResource method for JDK-8251377 without CSR

Reply:
@prsadhuk Thank you! Noted. Yes, it is the`getCellHighlightColorUIResource()` method.